### DISCUSSION TITLE
Hovering over functions

### DISCUSSION THREAD

**Discussion by Scooternut** [meerk40t/meerk40t#1847] [Q&A] [answered]
First of all, thanks for the amazing software you've developed, I certainly love the way it's been working for me and how it's evolved to be an awesome software that works incredibly. As a hobbyist and somewhat of a beginner with the K40 and your software, would it be possible to have the help/explanations that pop-up when you hover over them remain open rather than disappearing? Along with the videos that have been created, I'm really hungry for the information so that I can master using it.

**Comment by tatarize** [comment, marked as answer]
Easier said than done. The tool tips are done in the gui platform `wxPython` and while there are some settings for these. In settings `tooltip_autopop` and `tooltip_delay` can be manually set. `set tooltip_autopop 50000` in console, should force time for the tool tip to be higher than normal... but it doesn't work. You can set it really high and it doesn't stay any longer. You can apparently set very short periods or turn the tooltips off. But neither are actually helpful there.

**Reply by Scooternut** [reply]
Thank you very much for getting back to me! 
I can't imagine how complex this whole process of creating the program let alone this (minor) issue. I'll just keep doing what I've been doing, moving my mouse back and forth to be able to read them. And keep using scraps to tune to my different needs. 
Maybe at some point it'll work itself out, I think it was one of Newtons Law that states by every action there's a reaction 🤔 🤣😂.

**Reply by tatarize** [reply]
I'm not sure that's always the best there. They are sometimes a bit `meh` as far as descriptions go. If you find them confusing and wonder what they mean I'd be happy to make a video about the meaning of whatever settings you're finding confusion. The general rule is that if one person is confused then it's likely the case that more people are confused.

https://www.youtube.com/playlist?list=PLRcH9ggJLNEhDPPui1WEZwBb45P76Rh5X

**Reply by Scooternut** [reply]
I hear ya! Thanks for the offer, I'll definitely keep that in mind and promise not be a bother to you 😉. My knowledge is very weak when it comes to all the laser lingo, probably doesn't help (and my age doesn't help 😂). Planning on making it a priority to sit down and going through your videos one by one and following along and taking notes!

**Reply by Scooternut** [reply]
This is a little off topic (kinda 🤔), when playing the videos in "MeerK40t Instruction Videos", if you click "Play all", will it play all 38 of them in the correct order?

**Reply by tatarize** [reply]
There's not a correct order. They are mostly a bunch of random one off videos that tend to answer a particular thing I was specifically asked. So if you're wondering what some particular settings may do, it's usually easier for me to make a quick video than making a long bunch of text for a person describing what those things do.
